datapackage.json

```diff
@@ -1,116 +1,80 @@
 {
-  "licenses": [
+  "name": "global-temp",
+  "title": "Global Temperature Time Series",
+  "version": "0.1.0",
+  "license": "ODC-PDDL-1.0",
+  "sources": [
+    {
+      "name": "GISTEMP Global Land-Ocean Temperature Index",
+      "web": "http://data.giss.nasa.gov/gistemp"
+    },
     {
-      "name": "ODC-PDDL-1.0",
-      "path": "http://opendatacommons.org/licenses/pddl/",
-      "title": "Open Data Commons Public Domain Dedication and License v1.0"
+      "name": "Global component of Climate at a Glance (GCAG)",
+      "web": "https://www.ncei.noaa.gov/node/6696"
     }
   ],
-  "name": "global-temp",
   "resources": [
     {
-      "format": "csv",
-      "mediatype": "text/csv",
       "name": "annual",
       "path": "data/annual.csv",
+      "format": "csv",
+      "mediatype": "text/csv",
       "schema": {
         "fields": [
           {
             "name": "Source",
             "type": "string"
           },
           {
-            "description": "YYYY",
             "name": "Year",
-            "type": "year"
+            "type": "date",
+            "description": "YYYY"
           },
           {
-            "description": "Average global mean temperature anomalies in degrees Celsius relative to a base period. GISTEMP base period: 1951-1980. GCAG base period: 20th century average.",
             "name": "Mean",
-            "type": "number"
+            "type": "number",
+            "description": "Average global mean temperature anomalies in degrees Celsius relative to a base period. GISTEMP base period: 1951-1980. GCAG base period: 20th century average."
           }
         ]
       }
     },
     {
-      "format": "csv",
-      "mediatype": "text/csv",
       "name": "monthly",
       "path": "data/monthly.csv",
+      "format": "csv",
+      "mediatype": "text/csv",
       "schema": {
         "fields": [
           {
             "name": "Source",
             "type": "string"
           },
           {
-            "description": "YYYY-MM",
-            "format": "any",
             "name": "Date",
-            "type": "date"
+            "type": "date",
+            "description": "YYYY-MM"
```

Comment:
@gradedSystem please add `"format": "any"` as it is needed when performing data validation using this table schema.